### PR TITLE
New method for text tasks with TinyMCE [9.5.10]

### PR DESCRIPTION
**Pull request by nykroy**
After GLPI 9.5.10 Glpi had new method to print the textarea form with tinymce

### CODE REVIEW

**Comment by nykroy** [comment]
New method for text tasks with TinyMCE [9.5.10]

**Review by cedric-anne — APPROVED** [review]

File: inc/alert.class.php [hunk 1]
```diff
@@ -375,8 +375,15 @@ public function showForm($ID, $options = []) {
...
       echo '<td>' . __('Description') .'</td>';
       echo '<td colspan="3">';
-      echo '<textarea id="plugin_news_message_field" name="message" rows="12" cols="80" class="form-control">'.$this->getField('message').'</textarea>';
-      Html::initEditorSystem('plugin_news_message_field');
+      $content_id= "content$rand";
+      Html::textarea(['name'              => 'message',
+                         'value'             => $this->fields["message"],
+                         'rand'              => $rand,
+                         'editor_id'         => $content_id,
+                         'enable_fileupload' => false,
```
Comment (cedric-anne) [inline inc/alert.class.php:383]: This is the default behaviour.
```suggestion
```

File: inc/alert.class.php [hunk 2]
```diff
@@ -375,8 +375,15 @@ public function showForm($ID, $options = []) {
       echo '<tr>';
       echo '<td>' . __('Description') .'</td>';
       echo '<td colspan="3">';
-      echo '<textarea id="plugin_news_message_field" name="message" rows="12" cols="80" class="form-control">'.$this->getField('message').'</textarea>';
-      Html::initEditorSystem('plugin_news_message_field');
+      $content_id= "content$rand";
```
Comment (cedric-anne) [inline inc/alert.class.php:378]: `$rand` is not defined and this id is useless.
```suggestion
```

File: inc/alert.class.php [hunk 3]
```diff
@@ -375,8 +375,15 @@ public function showForm($ID, $options = []) {
...
+      Html::textarea(['name'              => 'message',
+                         'value'             => $this->fields["message"],
+                         'rand'              => $rand,
+                         'editor_id'         => $content_id,
```
Comment (cedric-anne) [inline inc/alert.class.php:382]:
```suggestion
```

File: inc/alert.class.php [hunk 4]
```diff
@@ -375,8 +375,15 @@ public function showForm($ID, $options = []) {
...
+                         'enable_fileupload' => false,
+                         'enable_richtext'   => true,
```
Comment (cedric-anne) [inline inc/alert.class.php:384]: Images should not be enabled, as they are not correctly transformed into documents.

```suggestion
                         'enable_richtext'   => true,
                         'enable_images'   => false,
```

**Comment by cedric-anne** [comment]
Manually merged in commit 44fffe3 after beiing adapted to GLPI 10.0.

GLPI 9.5 is no longer supported, but you can still apply this patch on your own GLPI instance.

Thanks for the contribution.

**Pull request closed without merging**